### PR TITLE
Use `flutter pub add` for dependencies in networking cookbooks

### PR DESCRIPTION
**Pull request by parlough**
This way users don't need to find the latest version themselves.

### CODE REVIEW

File: src/cookbook/networking/background-parsing.md [hunk 1]
```diff
@@ -38,9 +38,12 @@ First, add the [`http`][] package to your project.
 The `http` package makes it easier to perform network
 requests, such as fetching data from a JSON endpoint.
 
-```yaml
-dependencies:
-  http: <latest_version>
+To install the `http` package,
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```
Comment (atsansone) [inline src/cookbook/networking/background-parsing.md:43]: **suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To install the `http` package, run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```

File: src/cookbook/networking/delete-data.md [hunk 2]
```diff
@@ -23,13 +23,11 @@ This recipe uses the following steps:
 ## 1. Add the `http` package
 
 To install the `http` package,
-add it to the dependencies section of the `pubspec.yaml` file.
-You can find the latest version of the
-[`http` package][] on pub.dev.
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```
Comment (atsansone) [inline src/cookbook/networking/delete-data.md:27]: **suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```

File: src/cookbook/networking/fetch-data.md [hunk 3]
```diff
@@ -27,14 +27,12 @@ This recipe uses the following steps:
...
-dependencies section of the `pubspec.yaml` file.
-You can find the latest version of the
-[`http` package][] the pub.dev.
-
-```yaml
-dependencies:
-  http: <latest_version>
+To install the `http` package,
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```
Comment (atsansone) [inline src/cookbook/networking/fetch-data.md:32]: **suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To install the `http` package, run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```

File: src/cookbook/networking/send-data.md [hunk 4]
```diff
@@ -24,13 +24,12 @@ This recipe uses the following steps:
 
 ## 1. Add the `http` package
 
-To install the [`http`][] package, add it to the dependencies section
-of the `pubspec.yaml` file. You can find the latest version of the
-[`http` package][] on pub.dev.
+To install the `http` package,
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```
Comment (atsansone) [inline src/cookbook/networking/send-data.md:29]: **suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To install the `http` package, run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```

File: src/cookbook/networking/update-data.md [hunk 5]
```diff
@@ -26,14 +26,11 @@ This recipe uses the following steps:
...
-add it to the dependencies section
-of the `pubspec.yaml` file.
-You can find the latest version of the
-[`http` package][] on pub.dev.
-
-```yaml
-dependencies:
-  http: <latest_version>
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```
Comment (atsansone) [inline src/cookbook/networking/update-data.md:30]: **suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```

**Review by atsansone — COMMENTED** [review]
@parlough : Recommend updating to remove passive voice. Otherwise, LGTM.

**Comment by parlough** [comment]
Thanks for pointing me in that direction. I simplified the text and made it slightly more accurate.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**